facefusion/core.py

```diff
@@ -33,8 +33,20 @@
 
 
 def cli() -> None:
+	config.clear_config()
 	signal.signal(signal.SIGINT, lambda signal_number, frame: destroy())
-	program = ArgumentParser(formatter_class = lambda prog: HelpFormatter(prog, max_help_position = 160), add_help = False)
+	program = ArgumentParser(formatter_class=lambda prog: HelpFormatter(prog, max_help_position=160), add_help=False)
+    
+    # ini
+	program.add_argument('-i', '--ini', help=wording.get('help.ini'), dest='ini_path')
```

Comment:
the default should be 'facefusion.ini' so there is no need for most of the conditions - it cannot be empty